docs/activemq.md

```diff
@@ -0,0 +1,74 @@
+# `activemq` Metrics Receiver
+
+The `activemq` metrics receiver will be collected through the [Otel JMX Metric Gatherer](https://github.com/open-telemetry/opentelemetry-java-contrib/tree/main/jmx-metrics).
+.
+
+## Prerequisites
+
+JMX support must be enabled in the [broker configuration](https://activemq.apache.org/jmx).
+
+## Configuration
+
+| Field                 | Default            | Description |
+| ---                   |--------------------| ---         |
+| `endpoint`            | `localhost:1099`   | Must be in the form of `host:port`.|
+| `username`            | not set by default | The configured username if JMX is configured to require authentication. |
+| `password`            | not set by default | The configured password if JMX is configured to require authentication. |
+| `collection_interval` | `60s`              | A [time.Duration](https://pkg.go.dev/time#ParseDuration) value, such as `30s` or `5m`. |
+
+
+Example Configuration:
+
+```yaml
+metrics:
+  receivers:
+    activemq_metrics:
```

Comment:
It's preferred for the receiver and pipeline IDs to stay the same as the receiver type in the sample configuration.

aka `s/activemq_metrics/activemq`, `s/activemq_pipeline/activemq`